pages/db_views.py

```diff
@@ -0,0 +1,40 @@
+"""
+Views processing db based templates.
```

Comment:
Should we spawn new `View` entity? I believe that this will lead to ambiguity, because we will have two type of different things with the same name. I'd prefer to avoid this term. 

We may use this as part of context classes instead of db view or just use these classes without mentions about `View`

Up to you

Reply:
@artemiy312 , this class has the view nature. That's why every other name for this class will be bad.
It's not good to spawn the one other view module, i agree.
The best approach is to put the class to the existing view module. But we'll have problems with cyclic imports in this case.
So, new module name is just trade off to make django's standard design pleasure.

I'll add some of this words to the top of `db_view` module and leave it as is

Reply:
>this class has the view nature

But it isn't. A view is a callable which takes a request and returns a response. These classes don't match with this term

Reply:
@artemiy312 , y, you are right. So, we need some term for "template renderer with context".
I propose "Display". We already used it as part of property name `page.display_h1` for example.

I'll rename module